lk2nd/device/dts/msm8974/rules.mk

```diff
@@ -7,7 +7,6 @@ QCDTBS += \
 	$(LOCAL_DIR)/msm8974pro-ac-pm8941-mtp.dtb \
 	$(LOCAL_DIR)/msm8974-htc-m8.dtb \
 	$(LOCAL_DIR)/msm8974-lge-d855.dtb \
-	$(LOCAL_DIR)/msm8974-lge-hammerhead.dtb \
```

Comment:
I believe that hammerhead uses QCDTBS. Will check it later.

Reply:
It works with adtb I have checked

Reply:
Confirmed working. Thanks.